config_template/test_execution_config_template.h

```diff
@@ -40,12 +40,18 @@
  * #define TRANSPORT_INTERFACE_TEST_ENABLED  (0)
  */
 
-/** 
+/**
  * @brief Configuration to enable the OTA PAL test.
  *
  * #define OTA_PAL_TEST_ENABLED  (0)
  */
 
+/**
+ * @brief Configuration to enable the OTA End-to-end test.
+ *
+ * #define OTA_E2E_TEST_ENABLED  (0)
```

Comment:
What's the purpose of OTA_E2E_TEST_ENABLED?

Reply:
This tells the application to run MQTT agent and OTA task